com.unity.render-pipelines.universal/Runtime/UniversalRenderPipelineGlobalSettings.cs

```diff
@@ -175,6 +188,34 @@ void UpdateRenderingLayerNames()
             {
                 m_RenderingLayerNames[i] = string.Format("Unused {0}", i);
             }
+
+            // Update prefixed
+            if (m_PrefixedRenderingLayerNames == null)
+                m_PrefixedRenderingLayerNames = new string[32];
+            if (m_PrefixedLightLayerNames == null)
+                m_PrefixedLightLayerNames = new string[8];
+            for (int i = 0; i < m_PrefixedRenderingLayerNames.Length; ++i)
+            {
+                m_PrefixedRenderingLayerNames[i] = string.Format("{0}: {1}", i, m_RenderingLayerNames[i]);
+                if (i < 8)
+                    m_PrefixedLightLayerNames[i] = m_PrefixedRenderingLayerNames[i];
+            }
+        }
+
+        [System.NonSerialized]
+        string[] m_PrefixedLightLayerNames = null;
```

Comment:
You don't need the `= null` here, not really an issue.
See "Note no "= 0". All memory is zero'd out by default, so do not redundantly assign." in [C# ref](https://ono.unity3d.com/unity-extra/unity-meta/files/@/ReferenceSource/CSharp/Assets/CSharpReference.cs)

Reply:
good to know! thanks :)
do you prefer I rewrite the code or not?

Reply:
No, it isn't necessary.